lib/assert.js

```diff
@@ -131,8 +129,8 @@ function areSimilarTypedArrays(a, b) {
     }
     return true;
   }
-  return compare(from(a.buffer, a.byteOffset, len),
```

Comment:
I edited the comment above the function. I am not really sure what to write besides what I now changed.